src/docs/styles/docs-global.scss

```diff
@@ -56,12 +56,21 @@ main {
   pre[class*="language-"] {
     margin: var(--padding-3x) 0;
   }
+
+  table {
+    background: var(--card-background);
+  }
+
+  pre[class*="language-"] {
+    // We always want dark code block
+    background: #34294e;
```

Comment:
Can we use a variable here?

Reply:
I did not find any passing existing variable in light mode and did not wanted to add a variable in the lib as this code block is only meant for the doc showcase - should not exists in the lib. That's why I used a fix value and added a comment.